PE-739: Implement an option to use glob on upload-file command (#169)
ardriveapp/ardrive-cli: README.md

ardrive upload-file --local-file-path /path/to/folder  --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
```

> ### Uploading files using a glob pattern (BETA)

Reviewer: Thanks for adding this section! You'll also need to add a link to it in the table of contents above. Since the heading is rather long, use an anchor with an id so that you can easily link to it from the ToC, e.g. like the above:
```
### Uploading a Folder with Files (Bulk Upload)<a id="bulk-upload"></a>
```

Users can declare a glob pattern to select files to be upload in bulk. `upload-file` command will parse the glob pattern and will upload all the files that match the pattern to the selected destination.

> The example above will upload all `jpg` files from the indicated path.

Reviewer: `The example above` --> `The example below`

```shell

> ardrive upload-file --glob "/path/to/photos/folder/*.jpg" --parend-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json

Reviewer 2: spelling nit:

```suggestion
ardrive upload-file --glob "/path/to/photos/folder/*.jpg" --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
```

ardrive upload-file --local-file-path /path/to/folder  --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
```

### Uploading files using a glob pattern (BETA)<a id="glob-upload"></a>

> Users can declare a glob pattern to select files to be upload in bulk. `upload-file` command will parse the glob pattern and will upload all the files that match the pattern to the selected destination.

Reviewer: ```suggestion
Users can declare a glob pattern to select files to be uploaded in bulk. The `upload-file` command will parse the glob pattern and will upload all the files that match the pattern to a specified parent folder.
```

ardrive upload-file --local-file-path /path/to/folder  --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
```

### Uploading files using a glob pattern (BETA)<a id="glob-upload"></a>

Users can declare a glob pattern to select files to be uploaded in bulk. The `upload-file` command will parse the glob pattern and will upload all the files that match the pattern to a specified parent folder.
The example below will upload all `jpg` files from the indicated path.

```shell
ardrive upload-file --glob "/path/to/photos/folder/*.jpg" --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
```

> 

Reviewer: ```suggestion

NOTE: Ensure that the glob pattern is surrounded by quotations marks or unexpected upload behaviors or errors may occur.
```

Author: 👍
